app/Http/Controllers/Auth/ApplicationController.php

```diff
@@ -97,256 +72,60 @@ public function showApplicationForm(Request $request): View|RedirectResponse
-     * @return View
-     * @throws AuthorizationException
-     */
-    public function showApplications(Request $request): View
-    {
-        $authUser = $request->user();
-        if ($request->has('id')) { // return one application in detail
-            $user = User::withoutGlobalScope('verified')
-                ->with('application')->findOrFail($request->input('id'));
-            $this->authorize('view', $user->application);
-            return view('auth.application.applications_details', [
-                'user' => $user,
-            ]);
-        } else { //return all applications that can be visible
-            $this->authorize('viewSome', ApplicationForm::class);
-            if($authUser->can('viewAll', ApplicationForm::class)) {
-                $workshops = Workshop::all();
-            } else {
-                $workshops = $authUser->roleWorkshops->concat($authUser->applicationCommitteWorkshops);
-            }
-            $applications = ApplicationForm::select('*');
-            $applications->join('workshop_users', 'application_forms.user_id', '=', 'workshop_users.user_id');
-            if ($request->has('workshop') && $request->input('workshop') !== "null" && $workshops->contains($request->input('workshop'))) {
-                //filter by workshop selected
-                $applications->where('workshop_id', $request->input('workshop'));
-            } else {
-                //filter by accessible workshops
-                $applications->whereIn('workshop_id', $workshops->pluck('id'));
-            }
-            //hide unfinished
-            if ($authUser->cannot('viewUnfinished', \App\Models\ApplicationForm::class)) {
-                $applications->where(function ($query) {
-                    $query->where('status', ApplicationForm::STATUS_SUBMITTED)
-                        ->orWhere('status', ApplicationForm::STATUS_CALLED_IN)
-                        ->orWhere('status', ApplicationForm::STATUS_ACCEPTED);
-                });
-            }
-            //filter by status
-            if ($request->has('status')) {
-                $applications->where('status', $request->input('status'));
-            }
-            return view('auth.application.applications', [
-                'applications' => $applications->with('user.educationalInformation')->get()->unique()->sortBy('user.name'),
-                'workshop' => $request->input('workshop'), //filtered workshop
-                'workshops' => $workshops, //workshops that can be chosen to filter
-                'status' => $request->input('status'), //filtered status
-                'applicationDeadline' => $this->getDeadline(),
-                'periodicEvent' => $this->periodicEvent()
-            ]);
-        }
-    }
-
-    /**
-     * Edit an application's note.
-     * @param Request $request
-     * @return RedirectResponse
-     * @throws AuthorizationException
-     */
-    public function editApplication(Request $request): RedirectResponse
```

Comment:
This seems to be missing from AdmissionController.

Reply:
That is update in the AdmissionController